lib/QC_Counter.qpp

```diff
@@ -89,9 +89,13 @@ Counter::copy() {
     @code{.py}
 counter.inc();
     @endcode
+
+    @return the current value after the increment is returned
+
+    @since %Qore 0.8.13 this method returns the current value after the increment
```

Comment:
Will this be a part of Qore 0.8.13.9 or should this say 0.8.14 or maybe something else? Thanks.

Reply:
develop = 0.9